azkaban-db/src/main/sql/create.execution_flows.sql

```diff
@@ -19,4 +19,5 @@ CREATE INDEX ex_flows_start_time ON execution_flows(start_time);
 CREATE INDEX ex_flows_end_time ON execution_flows(end_time);
 CREATE INDEX ex_flows_time_range ON execution_flows(start_time, end_time);
 CREATE INDEX ex_flows_flows ON execution_flows(project_id, flow_id);
-CREATE INDEX executor_id ON execution_flows(executor_id);
\ No newline at end of file
+CREATE INDEX executor_id ON execution_flows(executor_id);
```

Comment:
I did not add this executor_id index. It's the same as the previous code. I guess it's showing as different because of the endline mark.